lib/mastodon/domains_cli.rb

```diff
@@ -28,10 +28,14 @@ def purge(domain)
         say('.', :green, false)
       end
 
-      DomainBlock.where(domain: domain).destroy_all
+      DomainBlock.where(domain: domain).destroy_all unless options[:dry_run]
 
       say
       say("Removed #{removed} accounts#{dry_run}", :green)
+
+      custom_emojis = CustomEmoji.where(domain: domain)
+      say("Removing #{custom_emojis.count} custom emojis", :green)
```

Comment:
The text above says "Removed x items" but this text says "Removing x items", I think it should be consistent?